scripts/addFromConfig.py

```diff
@@ -193,7 +193,7 @@ def addStuff(cfg, options):
         print(
             'Found Instrument: {0} {1}'.format(instrument_id,
                                                dbu.getEntry('Instrument', instrument_id).instrument_name))
-    except (DButils.DBNoData, ValueError, NoResultFound):
+    except (DButils.DBNoData, ValueError, AttributeError, NoResultFound):
```

Comment:
I'm not seeing any case where we'd get AttributeError here. This was connected to a bunch of the getEntry mess, so maybe that's why? Any reason you can think we'd need this?

Reply:
I'll try removing the AttributeError in the next round and make sure it still works.